defaults/main.yml

```diff
@@ -4,5 +4,9 @@
 # This file also serves as a documentation for such a variables.
 
 # Examples of role input variables:
-template_foo: foo
-template_bar: bar
+postgresql_version: "13"
+# postgresql_password: ""
```

Comment:
I think setting a value to an empty string is not a good idea. It is better to either set value to `null`, and then in tasks use `when: variable is not none`, or do not define a variable, and then use `when: variable is defined`.